styleguide/gulpfile.js

```diff
@@ -198,6 +199,10 @@ gulp.task('styles:inject', () => {
 gulp.task('styles:dist', (callback) => {
   _sassLint(false)
     .pipe(sourcemaps.init())
+    .pipe(sassdoc({
```

Comment:
I believe this should be in build.

As this gulpfile is currently being refactored, please create a seperate task for sassdoc and indicate where it should be used.

Reply:
Changed it to a separate task. I'll indicate in the PR for the gulp file refactoring where it should be used.